schema.graphql

```diff
@@ -226,7 +227,7 @@ type AcceptedOffer @entity {
   id: ID! # offerID:indexer
   indexer: Indexer!
   offer: Offer!
-  serviceAgreement: ServiceAgreement
+  serviceAgreement: ServiceAgreement!
```

Comment:
Might hit error at this part, when the insert speed not sync before the event trigger. Can keep in mind for this and validate later. @HZHAIX @lachlan1m